projects/packages/publicize/src/class-publicize-ui.php

```diff
@@ -664,36 +643,7 @@ class="wpas-submit-<?php echo esc_attr( $connection_data['service_name'] ); ?>"
 		</div>
 
 		<div id="pub-connection-needs-media"></div>
-
-		<?php if ( ! $all_done ) : ?>
```

Comment:
Similar point to above. Now will only display the message once the test connections call is complete. It's fine to remove it here and have the JS display it, but I think it might make sense to run that on page load.

Reply:
Yeah, we can improve that moving forward with connection test result cached.